### PR TITLE
lanes, new recipe

### PR DESCRIPTION
**Pull request by Begasus**
It builds and installs ok, test fails ...

### CODE REVIEW

File: dev-lua/lanes/lanes-3.11.recipe [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+SOURCE_URI="https://github.com/LuaLanes/lanes/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0a82fadeed96099b7bf90da92ece3862507a5e41a23778a7456b8adaf36e9180"
+PATCHES="lanes-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	lanes$secondaryArchSuffix = $portVersion
+	lib:core$secondaryArchSuffix
```
Comment (korli) [inline dev-lua/lanes/lanes-3.11.recipe]: This looks wrong. The library should be in $libDir/lua/5.3/lanes/, so no need to declare it.

Comment (Begasus) [reply dev-lua/lanes/lanes-3.11.recipe]: Ok, I'll try to look into it later :)

**Comment by Begasus** [comment]
luacheck still crashes with lanes installed "resolve symbol "pthread_yield" returned: -2147478780"

**Comment by Begasus** [comment]
Closing this one, I'll leave it in my own fork at github for historical if someone is looking for it :)

**Pull request closed without merging**